app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -75,82 +75,74 @@
       }
     };
 
+    var installTypes = {
+      android: 'gradle',
+      iOS: 'cocoapods',
+      cordova: 'npm'
+    };
+
+    var startServiceInstancesWatch = _.once(function(context) {
+      DataService.list(serviceClassesVersion, context, function(serviceClasses) {
+        serviceClasses = serviceClasses.by('metadata.name');
+        watches.push(DataService.watch(serviceInstancesVersion, context, function(serviceinstances) {
+          row.services = _.filter(serviceinstances.by('metadata.name'), function(serviceInstance) {
+            var serviceClass = _.get(serviceClasses, ServiceInstancesService.getServiceClassNameForInstance(serviceInstance));
+            return isMobileService(serviceClass) && isServiceInstanceReady(serviceInstance);
+          });
+          row.mobileClientEnabledServices = _.filter(row.services, function(service) {
+            var serviceClass = _.get(serviceClasses, ServiceInstancesService.getServiceClassNameForInstance(service));
+            return isMobileClientEnabled(serviceClass);
+          });
+          row.updateServicesInfo();
+        }, {errorNotification: false}));
+      });
+    });
+
+    var startBindingsWatch = _.once(function(context) {
+      watches.push(DataService.watch(serviceBindingsVersion, context, function(bindingsData) {
+        row.bindings = _.filter(bindingsData.by('metadata.name'), function(binding) {
+          return _.get(binding.metadata.annotations, 'binding.aerogear.org/consumer') === _.get(row, 'apiObject.metadata.name') && isBindingReady(binding);
```

Comment:
The `binding.aerogear.org/consumer` is added to the [annotationMap](https://github.com/aerogear/origin-web-common/blob/aerogear-mcp/src/filters/annotation.js#L35). We can use the annotationName filter to get this. 

```
mobileBindingConsumerID = $filter('annotationName')('mobileBindingConsumerId')
_.get(binding.metadata.annotations, mobileBindingConsumerID)
```